File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/contacts/EmailData.java
/*
 *  Copyright (c) Koninklijke Philips N.V., 2020
 *
 *  * All rights are reserved. Reproduction or dissemination
 *
 *  * in whole or in part is prohibited without the prior written
 *
 *  * consent of the copyright holder.
 *
 *
 */

package com.philips.cdp.prxclient.datamodels.contacts;

public class EmailData {
    private String label;
    private String contentPath;

    public void setLabel(String label) {
        this.label = label;
    }

    public void setContentPath(String contentPath) {
        this.contentPath = contentPath;
    }

    public String getLabel() {
        return label;
    }

    public String getContentPath() {
        return contentPath;
    }
}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/cdls/EmailData.kt
/*
 *  Copyright (c) Koninklijke Philips N.V., 2020
 *
 *  * All rights are reserved. Reproduction or dissemination
 *
 *  * in whole or in part is prohibited without the prior written
 *
 *  * consent of the copyright holder.
 *
 *
 */
package com.philips.cdp.prxclient.datamodels.cdls

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
class EmailData(var label: String? = null,
                var contentPath: String? = null
) : Parcelable